docs/sphinx/user_guide/feature/array.rst

```diff
@@ -0,0 +1,192 @@
+
+.. _array-label:
+
+======
+Arrays
+======
+
+Camp has its own Array implementation very similar to C++ std::array. Array functionality is
+included in the ``array.hpp`` file within ``include/camp/``. 
```

Comment:
The main difference, for both this and `tuple` is that the types are intentionally supported across host and device.  Might be worth mentioning.

Reply:
Done